bmwqemu.pm

```diff
@@ -211,18 +211,23 @@ sub set_ocr_rect {
 
 # util and helper functions
 
+sub get_timestamp {
+    my $t = gettimeofday;
+    return sprintf "%s.%04d ", (POSIX::strftime "%H:%M:%S", gmtime($t)), 10000 * ($t - int($t));
```

Comment:
Is UTC what we want to have in logs, or rather localized timezone? (`gmtime` vs `localtime`).